values/jobs/keycloak.gotmpl

```diff
@@ -67,5 +72,12 @@ nativeSecrets:
   TENANT_ID: {{ $v.oidc.tenantID }}
   TENANT_CLIENT_ID: {{ $v.oidc.clientID }}
   TENANT_CLIENT_SECRET: {{ $v.oidc.clientSecret }}
-script: {{ if $skipVerify }}NODE_TLS_REJECT_UNAUTHORIZED=0 {{ end }}npm run tasks:keycloak
+{{ if $hasStagingCerts }}
+files:
+  /tmp/node/certificates.crt: |
```

Comment:
Can we stick to Linux standards with `/etc/ssl/certs`?

Reply:
This is not for linux standards, it's for node. And I don't want to accidentally overwrite anything in `/etc/ssl/certs`

Reply:
It is confusing to me because node can mean a k8s node or nodejs, but I would still recommend to put certificates in `/etc/ssl/certs` or `/usr/share/local/extra`.

Reply:
I understand what you are saying, and under normal circumstances it is put there (using `_root-ca.tpl` script)
But here it needs to be stores in a tmp folder. Can rename to `/tmp/nodejs/certificates` if you prefer?

Reply:
fine by me